### PR TITLE
New Hierophant sprites

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: The Hierophant boss will now create an arena if you try to leave its arena.
imageadd: The Hierophant boss, its arena, and the weapon it drops all have new sprites.
soundadd: And new sounds.
wip: And new text.
/:cl:

rogue prison 2 2016-12-23 195002
mime edition hotel 2016-12-22 144237

### CODE REVIEW

**Comment by lzimann** [comment]
she's finally doing it!

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -759,28 +759,36 @@
 
 //Hierophant
 
-/obj/item/weapon/hierophant_staff
-	name = "Hierophant's staff"
+/obj/item/weapon/hierophant_club
+	name = "Hierophant's sclub"
```
Comment (Hatfish) [inline code/modules/mining/lavaland/necropolis_chests.dm]: >sclub

**Comment by WJohn** [comment]
Ask for fury's permission to use his arena sprites. They're very good.
https://tgstation13.org/phpBB/viewtopic.php?f=11&t=8104

For reference (NOT MY WORK):

**Comment by ExcessiveUseOfCobblestone** [comment]
I like the purple though :(

**Comment by ChangelingRain** [comment]
Those aren't purple and don't fit ANY of the theme for this!

**Comment by WJohn** [comment]
But this is super pretty and fits lavaland perfectly well.

**Comment by ExcessiveUseOfCobblestone** [comment]
make a demonic boss wj :^)

**Comment by AnturK** [comment]
I shed a tear of joy when i seen this PR.

**Comment by ChangelingRain** [comment]
It fits lavaland, but not the giant alien mech with a purple glowing club that the hierophant is going to end up being.

**Comment by Armhulen** [comment]
we could have it be tied to the abductors

the lore possibilities are endless

**Comment by jughu** [comment]
The battlefield that wjohn could be used for some devil boss maybe SATAN since all sins are on it

**Comment by Kromgar** [comment]
>Tells miners to fight it outside of the arena if you want to survive it

>now creates a new arena if you lure it out

Make up your mind

**Comment by anconfuzedrock** [comment]
This isn't fucking "sprites"

**Comment by ChangelingRain** [comment]
these are sprites, they're just not very good ones

**Pull request merged**